app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -325,6 +325,7 @@ alwaysDeny=Always deny
 alwaysAsk=Always ask
 autoHideMenuBar=Hide the menu bar by default
 disableTitleMode=Always show the URL bar
+wideURLbar=Use wide URL bar
```

Comment:
I actually am not sure if it sounds native. Should it be rather something like `Maximize the URL bar` or `Make the URL bar wide`, etc?